using-mobile-extensions/profile/profiles-event-reference.md

```diff
@@ -8,17 +8,25 @@ Here are the events that are handled by the Profile extension:
 
 The event is used to put a user profile attribute update action on the Event Hub. This will be consumed by the [User Profile](https://wiki.corp.adobe.com/display/ADMSMobile/V5+Module+-+User+Profile) module.
 
-This event will be generated iwhen an `updateUserAttribute` call is made by the SDK client.
+This event will be generated when an `updateUserAttribute` call is made by the SDK client.
+
+#### Event Details
+
+| **Event Type**                  | **Event Source**                     | **Paired** | **Paired Event** |
+| ------------------------------- | ------------------------------------ | ---------- | ---------------- |
+| com.adobe.eventType.userProfile | com.adobe.eventSource.requestProfile | No         |                  |
+
+
 
 #### Data Payload Definition
 
 Here are the key-value pairs in this event:
 
 | **Key** | **Value Type** | **Optional** | **Description** |
 | :--- | :--- | :--- | :--- |
-| userprofileupdatekey | Map&lt;String, Object&gt; | no | A map of all the attributes and the attribute values that the client wants to update/add to the User Profile that is managed by the Adobe Cloud Platform SDK. An existing attribute name will end up updating the attribute in the SDK, and a new attribute name will add a new attribute to the profile.The value of an attribute can be null, in which case the attribute will be deleted from the profileCurrently supported object types are String, Integer, Double, Boolean, Map and VectorAll the other types will be ignored while processing |
+| userprofileupdatekey | Map&lt;String, Object&gt; | no | A map of all the attributes and the attribute values that the client wants to update/add to the User Profile that is managed by the Adobe Cloud Platform SDK. An existing attribute name will end up updating the attribute in the SDK, and a new attribute name will add a new attribute to the profile.The value of an attribute can be null, in which case the attribute will be deleted from the profile. Currently supported object types are String, Integer, Double, Boolean, Map and VectorAll the other types will be ignored while processing |
```

Comment:
Editing changes: please update the "Description" to read as follows:

"A map of all the attributes and the attribute values that the client wants to update/add to the User Profile that is managed by the Adobe Cloud Platform SDK. An existing attribute name will update the attribute in the SDK, and a new attribute name will add a new attribute to the profile. When the value of an attribute is null, the attribute will be deleted from the profile. Currently supported object types are String, Integer, Double, Boolean, Map and Vector. All the other types will be ignored while processing."